docs/using_pydrofoil.md

```diff
@@ -0,0 +1,5 @@
+# Using pydrofoil
+
+Pydrofoil accepts a compiled RISCV exectuable and will run it. Typically, the
+source code of the executable will test some RISCV feature and report
+success/failure.
```

Comment:
Needs more work, either in this PR or maybe just merge as-is and extend in future enhancements?